wiki/Best_Of/en.md

```diff
@@ -1,283 +1,318 @@
-
 # Best Of series
 
 Index page of all the "Best Of" series.
 
+- [2009](./2009 "Best of 2009")
+- [2010](./2010 "Best of 2010")
+- [2011](./2011 "Best of 2011")
+- [2012](./2012 "Best of 2012")
+- [2013](./2013 "Best of 2013")
+- [2014](./2014 "Best of 2014")
+- [2015](./2015 "Best of 2015")
+- [2016](./2016 "Best of 2016")
+- 2017
+
 ### osu!standard
 
 #### 2009 osu!standard
```

Comment:
I am going to keep it.

I seriously do not want to scroll up/down just to check which gamemode is for this year in this line that I am modifying (sure as hell I will be pissed to redo if I moved standard to taiko and taiko to catch and then at mania realized I done it wrong).